server/src/main/java/org/opensearch/search/aggregations/metrics/AbstractPercentilesAggregationBuilder.java

```diff
@@ -86,9 +86,10 @@ public static <T extends AbstractPercentilesAggregationBuilder<T>> ConstructingO
          * out the behavior from there
          *
          * `args` are provided from the ConstructingObjectParser in-order they are defined in the parser.  So:
+         *
          *  - args[0]: values
          *  - args[1]: tdigest config options
-         *  - args[2]: hdr config options
+         *  - args[2]: hdr config options<
```

Comment:
```suggestion
         *  - args[2]: hdr config options
```

accidental?

Reply:
this it the last time ever that i do such a large PR.  What kind of magical GUI do you use to so easily spot mistakes ?